src/include/storage/index/hash_index.h

```diff
@@ -33,6 +33,8 @@ template<typename T>
 class DiskArray;
 class DiskArrayCollection;
 
+using visible_func = std::function<bool(common::offset_t)>;
```

Comment:
This is duplicated with the one in `local_hash_index.h`. Can we remove this one?

Reply:
Sure